akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -723,6 +725,7 @@ private final class Sdk(
     case e if e == classOf[Executor]           =>
       // The type does not guarantee this is a Java concurrent Executor, but we know it is, since supplied from runtime
       sdkExecutionContext.asInstanceOf[Executor]
+    case e if e == classOf[time.Clock] => time.Clock.systemUTC()
```

Comment:
This might be questionable, but I guess the Clock interface is a standard Java interface, so there is no point in reinventing it all the time for other entities.